### ISSUE TITLE
[Bug]: notification's collapsed_id is not working after update

### ISSUE THREAD

**Issue by jvojacek** [OneSignal/OneSignal-Android-SDK#1806]
### What happened?

Hello,
I updated the Android SDK version to [5.0.0, 5.99.99] and after this update the collapsed_id does not work. After receiving a notification with a unique collapsed_id, the old one (even with a unique_id) is replaced with the newer one.

When I downgrade back to version [4.0.0, 4.99.99], everything works correctly.

### Steps to reproduce?

```Markdown
1. install the OneSignal depencency implementation 'com.onesignal:OneSignal:[5.0.0, 5.99.99]'
2. Send notifications from the REST API or dashboard with a unique collapsed_id to the device.
3. receive the notification to the device
4. send another notification from the REST API or dashboard with a unique collapsed_id to the device.
5. the old notification is replaced with the newer one.
```

### What did you expect to happen?

I expected to see two notifications.

### OneSignal Android SDK version

[5.0.0, 5.99.99]

### Android version

13, 12, 11

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nan-li** [comment]
Hi @jvojacek, thank you for reporting, we will investigate.

**Comment by jennantilla** [comment]
@jvojacek this has been fixed in our latest Android release [5.0.2](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/5.0.2). Let us know if you have any other questions or concerns!
